gateway/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gateway"
-version = "0.12.0"
+version = "0.13.0"
```

Comment:
0.12.1

We don't need two major version bumps between this and the remaining work in #101.